test/action_view/preview_test.rb

```diff
@@ -2,10 +2,12 @@
 
 require "test_helper"
 
-class ActionView::PreviewTest < ActionView::Component::TestCase
+class ActionView::PreviewTest < ActionDispatch::IntegrationTest
```

Comment:
Maybe I can just move these into the integration_test but I miss rspec's 'context' blocks to organize tests.